src/server/stores/department-store.js

```diff
@@ -12,12 +12,16 @@ const fakeStore = {
   0: {
     id: 0,
     name: 'Safety',
-    lead: {
+    leads: [{
```

Comment:
i'm obviously killing this with fire as soon as i can

Reply:
🔥💀🔥